.github/workflows/tests.yml

```diff
@@ -1,87 +1,29 @@
 name: Tests
 
+# This uses workflows in a different repo, refer to:
+# https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow
+
+# This runs sailbot workspace workflows and actions.
+# Runs the Tests Definitions workflow in sailbot_workspace
+   
+# Run condition
 on:
   push:
     branches:
       - main
       - raye
   pull_request:
+  workflow_dispatch:
   schedule:
     # Every Friday at 1:00am UTC
     - cron: '0 1 * * 5'
-  workflow_dispatch:
-
-concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: true
```

Comment:
I'm not sure if concurrency should be defined here, in test_definitions.yml, or both. What do you think?

Reply:
Should be defined in the caller workflow